machines/Makefile

```diff
@@ -1,50 +1,56 @@
-find-dir-with = $(shell /usr/bin/perl -e 'chomp($$_ = `pwd`); while ($$_ ne "" && ! -e "$$_/$(1)") { m:(.*)/[^/]+/??:; $$_ = $$1; } print;')
-
-ifndef BSG_MANYCORE_DIR
-  export BSG_MANYCORE_DIR := $(call find-dir-with,.BSG_MANYCORE_ROOT)
-endif
+.DEFAULT_GOAL = all
 
+export BSG_MANYCORE_DIR := $(shell git rev-parse --show-toplevel)
```

Comment:
@vb000 I think this is über clever and super useful

Reply:
Do we still need these variables exported into the environment? Since we're no longer using file lists, it seems like we don't, and doing so is slightly dangerous.